### PR TITLE
Remove CSS from enqueue script

### PR DESCRIPTION
**Pull request by jonathanbardo**
### Description

- Remove CSS for extensions JS. The style was duplicated. 
- Move common.css into the style folder for clarity.

### CODE REVIEW

File: includes/class-coblocks-block-assets.php [hunk 1]
```diff
@@ -160,14 +160,6 @@ public function editor_assets() {
...
 		$filepath   = 'dist/' . $name;
 		$asset_file = $this->get_asset_file( $filepath );
-		$rtl        = ! is_rtl() ? '' : '-rtl';
-
-		wp_enqueue_style(
-			$name,
-			COBLOCKS_PLUGIN_URL . 'dist/style-' . $name . $rtl . '.css',
-			array(),
-			$asset_file['version']
-		);
```
Comment (AnthonyLedesma) [inline includes/class-coblocks-block-assets.php:170]: We are removing the RTL logic here? Why? We have RTL files generated and available to enqueue.

File: src/styles/index.scss [hunk 2]
```diff
@@ -68,6 +68,9 @@
 @import "../blocks/share/styles/editor.scss";
 @import "../blocks/social-profiles/styles/editor.scss";
 
+// @import "../formats/**/style.scss";
+@import "../formats/uppercase/style.scss";
```
Comment (AnthonyLedesma) [inline src/styles/index.scss:72]: Thank you for the implicit imports.

**Review by AnthonyLedesma — CHANGES_REQUESTED** [review]
Question about the removal of RTL enqueue.

File: includes/class-coblocks-block-assets.php (hunk 1 above)
Comment (olafleur-godaddy) [reply includes/class-coblocks-block-assets.php:170]: No, it's because the $rtl variable is calculated earlier already

Comment (AnthonyLedesma) [reply includes/class-coblocks-block-assets.php:170]: Oh you know what. I think I see why you did that now. We import that block_assets function and this editor_assets function.

Comment (AnthonyLedesma) [reply includes/class-coblocks-block-assets.php:170]: As a side note - We really need to refactor this file. We have this redundant import going on and it is very confusing to see what exactly is happening.

https://github.com/godaddy-wordpress/coblocks/blob/dcfae001f4b97cc050cda6d3e438b8e9e945631b/includes/class-coblocks-block-assets.php#L47-L49

**Review by AnthonyLedesma — APPROVED** [review]
Excellent. Working great!

File: includes/class-coblocks-block-assets.php (hunk 1 above)
Comment (jonathanbardo) [reply includes/class-coblocks-block-assets.php:170]: @AnthonyLedesma totally agree!

**Pull request merged**